3-transport/translations/Readme.hi.md

```diff
@@ -0,0 +1,24 @@
+# खेत से कारखाने तक परिवहन - खाद्य वितरण को ट्रैक करने के लिए आईओटी का उपयोग करना।
+
+कई किसान बेचने के लिए भोजन उगाते हैं - या तो वे वाणिज्यिक उत्पादक हैं जो अपने द्वारा उगाई गई हर चीज को बेचते हैं, या वे निर्वाह किसान हैं जो अपनी अतिरिक्त उपज को जरूरत की चीजें खरीदने के लिए बेचते हैं। किसी तरह भोजन को खेत से उपभोक्ता तक पहुँचाना पड़ता है, और यह आमतौर पर खेतों से थोक परिवहन द्वारा, हब या प्रसंस्करण संयंत्रों तक, फिर दुकानों पर भेजा जाता है। उदाहरण के लिए, एक टमाटर किसान टमाटर की कटाई करेगा, उन्हें बक्सों में पैक करेगा, बक्सों को ट्रक में लोड करेगा और फिर प्रसंस्करण संयंत्र को देगा। फिर टमाटरों को छांटा जाएगा, और वहां से उपभोक्ताओं को विभिन्न रूप में वितरित किया जाएगा।
```

Comment:
वाणिज्यिक can be replaced with व्यवसायिक for easy understanding

Reply:
It would be great if you find a better word for प्रसंस्करण for easy understanding. But if not, it's fine.

Reply:
How about it I add प्रोसेसिंग with this like प्रसंस्करण(प्रोसेसिंग) it will be helpfull